CONTACT

(512) 669-1318

Ann Arbor, MI

PROFILE

Bioinformatics scientist with 4+ years of experience developing high-performance workflows for NGS analysis, including data processing, curation, annotation, and statistical modelling. Experience in evolutionary phylogenomics, genome evolution, and SNV, CNV, and Fusion annotation R&D. Excels working independently and collaborating with cross-disciplinary teams. Experience mentoring academic and non-academic scientists.

EDUCATION

PhD in Computational Biology, 2019

The University of Texas at Austin [TX]

Masters in Plant Biology, 2013

State University of Campinas [Brazil]

Bachelor of Science, 2009

Federal University of Uberlandia [Brazil]

TECHNICAL SKILLS

System agnostic (Python, R, Bash, SQL Windows PowerShell)

Expertise in standard bioinformatics tools

Cloud computing (AWS SageMaker studio, Azure)

HPC (SLURM executor, batch processing)

Nextflow (SNV/CNV/Fusion variant call)

Generative AI prompting, RAG, embeddings

Statistical modeling and probability

Code documentation, version control (Git)

Team collab (Slack/Discord/Teams)

General data visualization (PowerBI)

Scientific communication and writing

EXPERIENCE

Bioinformatics Scientist at BioIVT, 2022-present

Company’s data person leading the local bioinformatics team and participating in global initiatives. Bridges the communication of data-driven biological insights between the company’s leadership and departments such as clinical data, lab operations, quality assurance and compliance, and global sales teams.

Current Projects

Implementing and deploying a data product that explores NGS, slide images, and clinical data assets

Proposing a new NGS biomarker annotation portfolio for FFPE and ctDNA samples

Exploring Generative AI to extract relevant information from free text in large clinical datasets

Collaborated on research exploring the use of FFPE blocks from lab-generated FNAs for molecular annotation (to be presented at Molecular TriCon, March 2025)

Past Projects

Developed an NGS pipeline that significantly reduced variant detection error, processing time, and ensured reproducibility of biospecimen biospecimen annotation

Postdoctoral Researcher, University of Michigan, 2020-2022

Conducted analyses to detect structural variants for evolutionary genomics (, ).

Past Projects

Developed and implemented  to assemble and analyze various NGS datasets, including transcriptomes, nuclear target capture, and organelle genomes

Collaborated on projects analyzing venom evolution in  and developing an  for NGS data processing

Developed a  to explore organelle genome variants

Leadership and service

Received multiple awards and grants that resulted in various

Instructed advanced bioinformatics at the  at  (2018-2022)

Dean's Council Student leader at the UT Austin College of Natural Sciences and the Dept. of Integrative Biology

Co-organized the Austin Chapter of PyLadies (Python meetup, 2018)