# **Deise JP GonÃ§alves**
**Bioinformatics | Computational Biology**  
ğŸ“ Ann Arbor, MI | ğŸ“§ deisejpg@gmail.com | ğŸ”— [LinkedIn](https://www.linkedin.com/in/deisegoncalves/) 
| ğŸ”— [GitHub](https://github.com/deisejpg) | ğŸŒ [Website](http://www.deisegoncalves.com/)  

---

## **ğŸ”¬ Profile**
Bioinformatics scientist with 4+ years of experience developing **high-performance workflows for 
NGS analysis**, including **data processing, curation, annotation, and statistical modeling**. 
Expertise in **phylogenomics, genome evolution, and SNV/CNV/Fusion annotation R&D**. Strong ability 
to work **independently and collaboratively** with cross-disciplinary teams. Experienced mentor for 
academic and industry scientists.

---

## **ğŸ“ Education**
- **PhD in Plant/Computational Biology**, The University of Texas at Austin *(2019)*
- **MSc in Plant Biology**, State University of Campinas, Brazil *(2013)*
- **BSc in Science**, Federal University of Uberlandia, Brazil *(2009)*

---

## **ğŸ›  Technical Skills**
âœ… **Programming:** Python, R, Bash, SQL, PowerShell  (system agnostic)  
âœ… **NGS Analysis:** SNV, CNV, Fusion detection, Variant Annotation and standard bioinformatics tools  
âœ… **Cloud Computing:** AWS SageMaker, Azure, HPC (SLURM, batch processing)  
âœ… **Workflow Automation:** Nextflow, Docker  
âœ… **Machine Learning & AI:** Generative AI, RAG, Statistical Modeling  
âœ… **Data Visualization:** Power BI, Matplotlib, ggplot2  
âœ… **Version Control & Collaboration:** Git, Slack, Teams

---

## **ğŸ’¼ Experience**

### **Bioinformatics Scientist** | *BioIVT* *(2022â€“Present)*  
Companyâ€™s data person leading the local bioinformatics team and participating 
in global initiatives. Bridges the communication of data-driven biological insights 
between the companyâ€™s leadership and departments such as clinical data, lab operations, 
quality assurance and compliance, and global sales teams.  

#### **ğŸ“‚ Current Projects**
- Implementing and deploying a data product that explores NGS, slide images, and clinical data assets
-	Proposing a new NGS biomarker annotation portfolio for FFPE and ctDNA samples 
-	Exploring Generative AI to extract relevant information from free text in large clinical datasets
-	Collaborated on research exploring the use of FFPE blocks from lab-generated FNAs for molecular 
annotation (to be presented at Molecular TriCon, March 2025)  

#### **ğŸ“‚ Past Projects**
- Developed an NGS pipeline that significantly reduced variant detection error, processing time, 
and ensured reproducibility of biospecimen biospecimen annotation

### **Postdoctoral Researcher** | *University of Michigan* *(2020â€“2022)*
Conducted analyses to detect structural variants for evolutionary genomics 
([NSF project](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1917146&HistoricalAwards=false), 
[publication](https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.20200)).  

-	Developed and implemented [various pipelines](https://github.com/deisejpg/manipulating_fastas.git)
to assemble and analyze various NGS datasets, including transcriptomes, nuclear target capture, 
and organelle genomes 
-	Collaborated on projects analyzing venom evolution in 
[snakes](https://github.com/deisejpg/mito-genes_from_transcriptome) 
and developing an [analytical tool](https://github.com/edgardomortiz/sam2consensus) 
for NGS data processing
-	Developed a [method](https://github.com/deisejpg/rosids) to explore organelle genome variants

---

## **ğŸ¤ Leadership, Service & Teaching**

**Instructor**, taught bioinformatics at the 
[Workshop on Molecular Evolution](https://molevolworkshop.github.io/about/) at 
[MBL](https://molevolworkshop.github.io/?_ga=2.119272586.2013273605.1651511791-1963788801.1650375037) *(2018â€“2022)*  
**Organizer**, Austin Chapter of PyLadies Python Meetups *(2018)*  
**Reviewer**, Scientific Journals & Conference Proceedings  
**Student leader**, Dean's Council Student leader at the UT Austin College of Natural Sciences 
and the Deptment of Integrative Biology 
ğŸ†Received multiple awards and grants that resulted in various [publications](https://scholar.google.com/citations?user=l0q5YpsAAAAJ&hl=en)
